lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -99,6 +99,23 @@ def self.tag_create(category, options = {})
       end
     end
 
+    def self.tag_delete(category, entry, force_delete = false)
```

Comment:
My suggestion for the `force_delete` flag was a thought on what we might **consider in the future** if we find there is enough need for it.  I suggest we do not expose that at this time and wait until there is a clear use case for it.

Reply:
Sounds good to me. Since it is not hard to add this flag, I will remove it from this PR now and wait for more use cases.